src/sentry/models/savedsearch.py

```diff
@@ -19,6 +19,7 @@ class SortOptions:
     USER = "user"
     TREND = "trend"
     INBOX = "inbox"
+    BETTER_PRIORITY = "betterPriority"
```

Comment:
I didn't bother feature flagging this since you can't even get to the point of saving a search if you can't select it (which is feature flagged).